Jenkinsfile

```diff
@@ -1,11 +1,56 @@
 @Library('jenkins-library')
 
-def pipeline = new org.js.AppPipeline(
-    steps:              this,
-    test:               false,
-    dockerRegistryCred: 'bot-sora2-rw',
-    dockerImageName:    'sora2/subquery',
-    buildDockerImage:   'docker.soramitsu.co.jp/build-tools/node:14-ubuntu',
-    preBuildCmds:       ['yarn install', 'yarn codegen']
-)
```

Comment:
You broke the CI for the app, but should have created a new Jenkisfile-onfinality file